docs/commands/test.md

```diff
@@ -59,6 +59,10 @@ empty, Modular will write a message to `stdout` and exit with code `0`.
 
 ### Modular-specific options
 
+`--jest`: Bypass Modular selective behaviour and flags, and send all provided
```

Comment:
My proposals:
- `--bypass`
- `--bypassToJest`

Reply:
bypass sounds good to me me